core/modules/compare/components/Compare.ts

```diff
@@ -0,0 +1,47 @@
+import i18n from '@vue-storefront/i18n'
+import Product from '@vue-storefront/store/types/product/Product'
+
+export const Compare = {
+  name: 'Compare',
+  props: {
```

Comment:
I don't think the title should be determined by a prop. Can you take it from products info? Moving to modules also involves correcting poor APIs like this

Reply:
This is prop which comes from routing. I moved it to theme with metaData object, b/c it's related.

Reply:
damn, you're right. I somehow missed that it's a prop